tools/wagmi-example/src/App.tsx

```diff
@@ -0,0 +1,35 @@
+/*-
+ *
+ * Hedera JSON RPC Relay - Wagmi Example
+ *
+ * Copyright (C) 2022-2024 Hedera Hashgraph, LLC
```

Comment:
nit: new files should be fine with just the current year I believe.
```suggestion
 * Copyright (C) 2024 Hedera Hashgraph, LLC
```